pkg/console/operator/operator.go

```diff
@@ -238,16 +238,21 @@ func (c *consoleOperator) handleSync(configs configSet) error {
 	// set default conditions ok first, sync can toggle if conditions are invalid
 	c.ConditionsDefault(updatedStatus)
 	err := c.sync_v400(updatedStatus, configs)
-	if err != nil {
-		if !customerrors.IsSyncError(err) || !customerrors.IsCustomLogoError(err) {
-			c.SyncStatus(c.ConditionResourceSyncDegraded(updatedStatus, err.Error()))
+
+	c.HandleDegraded(updatedStatus, "SyncError", func() error {
```

Comment:
These handler can't be hidden behind `if` statements, else we will be pinned in a permanent condition.  The handler deals with both `true` & `false`.  Thus this block of logic needs to change a bit.

Reply:
It kinda dost but this pretty good case to use it :)